frontend/src/app/app.component.html

```diff
@@ -0,0 +1,15 @@
+<h2>File Upload</h2>
+<input type="file" (change)="onFileSelected($event)">
+
+<h2>Test Get Request</h2>
+<form [formGroup]="form">
+  <div>
+    <label>SourceCode:</label>
+    <input type="text" formControlName="sourceText">
+  </div>
+  <div>
+    <label>Language:</label>
+    <input type="text" formControlName="language">
```

Comment:
This should probably be a `<select>`

Reply:
Otherwise we can not test the error case